app/resources/en/translation/screeners/starteligibility.json

```diff
@@ -1,37 +1,11 @@
-  "paragraph7": "The application fee is based on the value of the estate. There&rsquo;s no fee if the estate is &pound;50,000 or under.",
-  "paragraph8old": "Extra copies of the grant of probate cost &pound;1.50 each.",
-  "paragraph8": "Extra copies of the grant of probate cost &pound;10 for the first copy and 50 pence for every copy after that.",
-  "paragraph9": "You&rsquo;ll need to pay online by debit or credit card.",
-  "tableHeadLeft": "Estate Value",
-  "tableHeadRight": "Fee",
-  "tableBodyFeeRange1": "&pound;50,000 or under",
-  "tableBodyFeeRange1Value": "Free",
-  "tableBodyFeeRange2": "Over &pound;50,000 to &pound;300,000",
-  "tableBodyFeeRange2Value": "&pound;250",
-  "tableBodyFeeRange3": "Over &pound;300,000 to &pound;500,000",
-  "tableBodyFeeRange3Value": "&pound;750",
-  "tableBodyFeeRange4": "Over &pound;500,000 to &pound;1 million",
-  "tableBodyFeeRange4Value": "&pound;2,500",
-  "tableBodyFeeRange5": "Over &pound;1 million to &pound;1.6 million",
-  "tableBodyFeeRange5Value": "&pound;4,000",
-  "tableBodyFeeRange6": "Over &pound;1.6 million to &pound;2 million",
-  "tableBodyFeeRange6Value": "&pound;5,000",
-  "tableBodyFeeRange7": "Over &pound;2 million",
-  "tableBodyFeeRange7Value": "&pound;6,000",
-  "feedbackHeader": "If you have feedback",
-  "paragraph10": "We&rsquo;re currently building this service. If you notice things that don&rsquo;t work as well as they should, you can use the &lsquo;feedback&rsquo; link at the top of the page.",
-  "paragraph11": "Return to an existing application"
+  "paragraph1": "You will need to submit an application and send your documents to the probate registry. We&rsquo;ll tell you exactly what you need to send once you have completed your application.",
+  "paragraph2": "<dl class=\"inline-dl\"><dt>You can apply in Welsh by post. </dt><dd>Use <a href=\"{applicationFormPA1P}\" class=\"govuk-link\" target=\"_blank\" rel=\"external noopener\" aria-label=\"Form PA1P\">Form PA1P</a> if the person who died left a will, or <a href=\"{applicationFormPA1A}\" class=\"govuk-link\" target=\"_blank\" rel=\"external noopener\" aria-label=\"Form PA1A\">Form PA1A</a> if there is no will.</dd></dl>",
+  "multipleHeader": "What you will need",
+  "paragraph3": "Before you start your application, you will need:<br><ul class=\"govuk-list govuk-list--bullet\"><li>the details of the person who died</li><li>the original will, if the person who died left one, and any codicils</li><li>the name, email address and mobile phone number for each person applying for probate</li><li>the values of the estate for Inheritance Tax (IHT) – you can estimate these using HMRC&rsquo;s <a href=\"{ihtChecker}\" rel=\"external noopener\" target=\"_blank\" class=\"govuk-link\" aria-label=\"online IHT checker\">online IHT checker</a></li><li>a debit or credit card, if you need to pay a fee</li></ul>",
```

Comment:
Shouldn't the markup here be in the template with the content (the strings here) being rendered into the template rather than having the structure of the document defined by the content?

Reply:
Yes you are right.It is better to keep structure in HTML itself.Implemented for this change.